common4j/src/main/com/microsoft/identity/common/java/flighting/CommonFlight.java

```diff
@@ -98,7 +98,13 @@ public enum CommonFlight implements IFlightConfig {
     /**
      * Flight to enable the attachment of PRT header in cross cloud requests. Default is true.
      */
-    ENABLE_ATTACH_PRT_HEADER_WHEN_CROSS_CLOUD("EnableAttachPrtHeaderWhenCrossCloud", true);
+    ENABLE_ATTACH_PRT_HEADER_WHEN_CROSS_CLOUD("EnableAttachPrtHeaderWhenCrossCloud", true),
+
+    /**
+     * Flight to enable adding x-client-MN and x-client-WPAvailable extra query parameters
+     */
+    ENABLE_AM_API_WORKPROFILE_EXTRA_QUERY_PARAMETERS("EnableAmApiWorkProfileExtraQueryParameters", true);
```

Comment:
Is the default value for this flight intended to be true?

Reply:
Yes, we want the parameters to be sent by default, we just have the flight in order to turn it off if needed